WcaOnRails/app/controllers/api/internal/v1/api_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Api::Internal::V1::ApiController < ApplicationController
+  prepend_before_action :validate_token
+
+  def validate_token
+    service_token = request.headers["X-WCA-Service-Token"]
```

Comment:
Consider moving the header name into a constant somewhere

Reply:
I guess it should be the microservice_helper that I made in the other PR?

Reply:
Sounds like a good place for it!

Reply:
We still haven't decided if we should put it in lib or helpers though! Considering it contains routes which might be used in views or helpers...